src/screens/StoryScreen.tsx

```diff
@@ -1,10 +1,42 @@
-import React from 'react';
-import { Text, View } from 'react-native';
+import React, { useEffect, useState } from 'react';
+import { ActivityIndicator, ScrollView, View } from 'react-native';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
I would remove this disable comment, as the issue you were seeing in your code is just VSCode bugging. You have react-native-htmlview in your package.json so it will pass eslint checks.